src/styles/antisnake.scss

```
@@ -0,0 +1,37 @@
+.antisnake {
+  @include grid-centered;
+  grid-template-columns: repeat(3, 1fr);
+
+  &__block {
+    @include flex-centered;
+    @include antisnake__block-size;
+
+    color: white;
+    &:nth-child(1) {
+      background-color: #ff0102;
+    }
+    &:nth-child(2) {
+      background-color: #cc0202;
+    }
+    &:nth-child(3) {
+      background-color: #9a0002;
+    }
+    &:nth-child(4) {
+      background-color: #670100;
+    }
+    &:nth-child(5) {
+      background-color: #340000;
+    }
+    &:nth-child(6) {
+      background-color: #000;
+    }
+  }
+
+  @media screen and (max-width: 750px) {
+    grid-template-columns: repeat(2 , 1fr);
+  }
+  
+  @media screen and (max-width: 450px) {
+    grid-template-columns: repeat(1 , 1fr);
+  }
```

Навіщо використовувати repeat?
```suggestion
  @media screen and (max-width: 450px) {
    grid-template-columns: 1fr;
  }
```

---

копіпаст з попереднього медіазапиту :) виправив